lib/notifications/db/detour_expiration.ex

```diff
@@ -37,11 +37,17 @@ defmodule Notifications.Db.DetourExpiration do
   Creates a new "Detour Expiration Notification"
   """
   def changeset(detour_expiration, attrs) do
+    params =
+      attrs
+      # A `%DetourExpiration{}` should always have a associated notification, so
+      # add a placeholder to the params in case `attrs` does not contain it.
+      |> Map.put_new(:notification, %{})
+      # All users are given a detour expiration notification at this time,
+      # so it is configured as default in the changeset
+      |> put_in([:notification, :users], Skate.Settings.User.get_all())
+
```

Comment:
note(request-for-feedback): I'm not sure that this logic belongs here vs `notification.ex` so open to any reasons why it should be moved.